assets/css/tasks/_article-tasks.css

```diff
@@ -0,0 +1,114 @@
+/* SINGLE ARTICLE PAGE */
+.container {
+    font-family: 'Lato', sans-serif;
+}
+
+.post-page__img { 
+    width: 100%;
+    max-height: 250px;
+    object-fit: contain;
+    overflow: hidden;
+    object-fit: cover;
+    padding: 0px 250px 0px 250px;
+    margin-top: 50px;
+}
+
+.post-page__header {
+    width: 100%;
+    padding-bottom: 100px;
+}
+
+.u-margin--medium {
+    margin: 100px 350px 50px 350px;
+}
+
+.post-description__title {
+    float: left;
+    margin-bottom: 30px;
+}
+
+.post-description__date {
+    float: right;
+}
+
+.post-end {
+    margin: 100px 300px 50px 300px;
+}
+
+.post-page-end__button--previous,
+.post-page__author-info,
+.post-page-end__button--next {
+    float: left;
+    width: 33%;
+    /* width; all three elements at the end of the page should share the available width between them, so one element should take up 1/3 of the entire available width*/
+}
+
+.post-page-end__button--next {
+    float: right;
+    color: #7a24ab;
+    text-align: right;
+    font-weight: bold;
+    padding-top: 35px;
+}
+
+.post-page-end__button--previous { 
+    text-align: left; 
+    color: #7a24ab;
+    font-weight: bold;
+    padding-top: 35px;
+}
+
+.arrow {
+    border: solid #7a24ab;
+    border-width: 0 3px 3px 0;
+    display: inline-block;
+    padding: 3px;
+  }
+  
+.right {
```

Comment:
Whoow, you made the transformation! I'm so glad to see it.
I just want to give you a tip on naming a class of selectors. The single word "arrow" is meaningfull, but in my opinion "left" and "right" without context, it is not. To correct this, you can rename them to "arrow-left" and "arrow-right".
You have a good intuition that repetition even in naming is wrong, but with pure CSS we don't have much to do. This is where the modern frame comes in. For example, "SASS" is a framework where you can reuse names, and it will look just like you did.

So SASS or some other CSS framework might be your next step;)